README.md

```diff
@@ -1,16 +1,23 @@
-# Bazel Docker Rules
+# Bazel Container Image Rules
```

Comment:
Just "Bazel Container Rules" ?

Reply:
This doesn't do stuff like Kubernetes, so I scoped it to images.